pages/video-download/video-download.view.html

```diff
@@ -1,5 +1,18 @@
 <div ng-controller="VideoDownloadController as VideoDownload" class="download-page">
-	<article>
+
+	<!-- Screen loading directive -->
+	<div ng-if="!VideoDownload.video.data && VideoDownload.loading" class="loading">
```

Comment:
maybe a 404 error directive here for handling such 404 backend errors and having consistent errors accross all the platform?

Reply:
I don't know if we shall have one only 404 error page, or have different ones for each type of content (such as, for example, [youtube does](https://www.youtube.com/watch?v=oTugqOT0))

Reply:
lets talks about later with pablo and juan, two approaches are commonly seen in other platforms